src/main/java/dev/arxiv/name/mappers/SubjectClassificationMapper.kt

```diff
@@ -0,0 +1,161 @@
+package dev.arxiv.name.mappers
+
+private val classificationMap: Map<String, String> = mapOf(
+        Pair("astro-ph", "Astrophysics"),
```

Comment:
Can we load it from someone resource ?

Reply:
Only if we will be try to parse the arxiv.org api documentation. I suppose, it's not reliable decision for software library.